.github/workflows/main.yml

```diff
@@ -20,6 +20,13 @@ jobs:
       run: ./scripts/bluepill.sh test
     - name: Build Bluepill
       run: ./scripts/bluepill.sh build
+    - name: Release
+      uses: softprops/action-gh-release@v1
+      if: github.event_name	== 'push' && startsWith(github.ref, 'refs/tags/release-v')
```

Comment:
I think this should be `refs/tags/v`, I did a `git fetch --all --tags` and then
```
% ls .git/refs/tags
1.1.0   3.0.0   9.0.0   v0.1.0  v0.2.1  v0.2.3  v1.0.1  v1.1.1  v2.0.0  v2.0.2  v2.2.0  v2.3.1  v3.0.1  v3.1.0  v4.0.0  v4.1.1  v5.0.0
2.4.0   3.1.0   list    v0.2.0  v0.2.2  v1.0.0  v1.0.2  v1.1.2  v2.0.1  v2.1.0  v2.3.0  v3.0.0  v3.0.2  v3.1.1  v4.1.0  v4.2.0```

Reply:
Yup!